testsuite/N806_py3.py

```diff
@@ -0,0 +1,17 @@
+# python3 only
+#: Okay
+VAR1, *VAR2, VAR3 = 1, 2, 3
+#: Okay
+[VAR1, *VAR2, VAR3] = (1, 2, 3)
+#: N806
+def extended_unpacking_ok():
```

Comment:
This name seems to imply that the following is ok but we're verifying we get an N806 so I'm confused which  it is we're expecting